src/Kernel/CachedContainerBuilder.php

```diff
@@ -72,7 +72,7 @@ public function clearCache(): void
             return;
         }
 
-        $smartFileSystem = new SmartFileSystem();
-        $smartFileSystem->remove($cacheFiles);
+        $filesystem = new Filesystem();
+        $filesystem->remove($cacheFiles);
```

Comment:
@staabm I am wondering why remove cache container called in normal `vendor/bin/rector` on as should only run on phpunit run?